blueocean-analytics-tools/src/main/resources/io/jenkins/blueocean/analyticstools/AnalyticsTools/header.jelly

```diff
@@ -1,32 +1,34 @@
-          ]]>
+                  return normalizedUrl;
+                }
+                var transformer = function(payload) {
+                  payload.data.request.user_ip = '0.0.0.0';
+                  payload.data.request.url = normalizeURL(window.location.pathname);
+                };
+                var _rollbarConfig = {
+                  accessToken: "81f3134dedf44871b9cc0a347b1313df",
+                  captureUncaught: true,
+                  code_version: "${it.blueOceanPluginVersion}",
+                  source_map_enabled: true,
+                  guess_uncaught_frames: true,
+                  transform: transformer
+                };
+                <![CDATA[
```

Comment:
@tfennelly Ugh! Do we really need, it. By default it doesn't appear in index.html, to get in index.html you have to pass a JVM property set to true. That is its going to be enabled on dogfood server to report errors. So not in critical path. So unless you tell me its going to interfere with functioning of other things, I intend to keep it as it is.